iree_tests/pytorch/models/llama-tank/test_cases.json

```diff
@@ -0,0 +1,18 @@
+{
+  "test_cases": [
+    {
+      "name": "splats",
+      "runtime_flagfile": "splat_data_flags.txt",
```

Comment:
> On that note, looks like splat test cases aren't running at the moment?

The test code is filtering to just real weight tests (using `-k real_weights`) right now:

https://github.com/nod-ai/SHARK-TestSuite/blob/73dcb4239cea621e4d5366b04f2bde7ffccd4470/.github/workflows/test_iree.yml#L75-L78

I consider the `test_iree.yml` in this repo to be a sanity check and demonstration of the test suite itself, not a source of truth for test coverage. https://github.com/openxla/iree/pull/16801 adds similar tests in the IREE repo, and that is what "really matters" (will run on presubmit and should be kept passing). To that end, we can add whatever new filtering and test cases we want here.

Reply:
Got it! Thanks for the info.

Reply:
Speaking of how brittle the tests are in this repo (by using the latest IREE nightlies instead of a pinned version), you may want to sync past https://github.com/nod-ai/SHARK-TestSuite/commit/28091b0996239f411df5850b1a7e41875ae047d2 to get a clean CI run on this PR